runtime/onert/core/src/exec/train/TrainableExecutor.cc

```diff
@@ -195,8 +195,16 @@ float TrainableExecutor::getLoss(const ir::IOIndex &pred_io_ind) const
   if (loss_ind.undefined())
     throw std::runtime_error{"Loss " + std::to_string(loss_ind.value()) + " is not defined."};
   backend::ITensor *tensor = _tensor_regs.getITensor(loss_ind);
-  auto loss_buf = reinterpret_cast<float *>(tensor->buffer());
-  return *loss_buf;
+  double sum = 0.f;
```

Comment:
(Optional) According to https://en.cppreference.com/w/cpp/language/floating_literal, the suffix "f" means `float`. For the literal meaning double, "f" must be removed such as "0.".
But there is no problem with the suffix "f".